config.go

```diff
@@ -164,20 +164,28 @@ func Load(name string) *Config {
 // Finish is the last function to be exexuted by any configuration to output
 // details to the user.
 func (c *Config) Finish() {
+	ansiYellow := "\033[30;48;5;226m"
```

Comment:
Shouldn't these match the colors for Waterlog?

See: https://github.com/DataDrake/waterlog/blob/master/format/style.go

Reply:
Colors have been fixed

Reply:
Should the skipped configurations be warning or info lines?